tools/sof-kernel-log-check.sh

```diff
@@ -246,32 +246,21 @@ case "$platform" in
         ;;
 esac
 
-[[ ! "$err_str" ]] && echo "Missing error keyword list" && exit 0
-# dmesg KB size buffer size
-#dmesg_config_define=$(awk -F '=' '/CONFIG_LOG_BUF_SHIFT/ {print $2;}' /boot/config-$(uname -r))
-#dmesg_buffer_size=$( echo $(( (1<<$dmesg_config_define) / 1024 )) )
-# kernel file log buffer size
-#kernel_buffer_size=$(du -k /var/log/kern.log |awk '{print $1;}')
-# now decide using which to catch the kernel log
-#[[ $kernel_buffer_size -lt $dmesg ]] && cmd="dmesg" || cmd="tail -n  +${begin_line}  /var/log/kern.log"
+# confirm begin_timestamp is in UNIX timestamp format, otherwise use dmesg
```

Comment:
Please add a comment explaining why we still need  to support a different begin_timestamp format.

Reply:
I do not think we need **a different begin_timestamp format.** What I mean here is to keep the fallback we used to have to dmesg now to cover the small migration steps.